test/js/style/style.test.js

```diff
@@ -560,17 +575,6 @@ test('Style#featuresAt', function(t) {
             });
         });
 
-        t.test('includes arbitrary keys', function(t) {
```

Comment:
This is from #910. If arbitrary layer properties is still a necessary feature (@edenh?) we need to make validation permit them.